--- vkeybd.orig/Makefile	2007-08-28 15:05:17.000000000 +0200
+++ vkeybd/Makefile	2007-11-30 20:13:44.000000000 +0100
@@ -8,7 +8,7 @@
 #
 # installation directory
 #
-PREFIX = /usr/local
+PREFIX = /usr
 # binary and Tcl script are put there
 BIN_DIR = $(PREFIX)/bin
 # man page
@@ -36,9 +36,9 @@
 # Tcl/Tk library -- depends on your distribution
 #
 TCL_VERSION = 8.4
-TCLLIB = -ltcl$(TCL_VERSION)
+TCLLIB = -ltcl
 TCLINC =
-TKLIB = -ltk$(TCL_VERSION)
+TKLIB = -ltk
 TKINC =
 XLIB = -L/usr/X11R6/lib -lX11
 XINC = -I/usr/X11R6/include
@@ -84,7 +84,7 @@
 
 VKB_TCLFILE = $(VKBLIB_DIR)/vkeybd.tcl
 
-COPTFLAGS = -Wall -O
+COPTFLAGS = -Wall
 CFLAGS = $(COPTFLAGS) -DVKB_TCLFILE=\"$(VKB_TCLFILE)\" \
 	-DVKBLIB_DIR=\"$(VKBLIB_DIR)\"\
 	-DVERSION_STR=\"$(VERSION)\"\
