diff -aurN vkeybd.orig/Makefile vkeybd/Makefile
--- vkeybd.orig/Makefile	2003-01-23 16:41:33.000000000 +0100
+++ vkeybd/Makefile	2003-07-19 17:17:09.000000000 +0200
@@ -8,7 +8,7 @@
 #
 # installation directory
 #
-PREFIX = /usr/local
+PREFIX = /usr
 # binary and Tcl script are put there
 BIN_DIR = $(PREFIX)/bin
 # man page
@@ -27,7 +27,7 @@
 USE_AWE = 1
 USE_MIDI = 1
 USE_ALSA = 1
-USE_LADCCA = 0
+USE_LADCCA = 1
 
 #
 # Tcl/Tk library -- depends on your distribution
@@ -81,7 +81,8 @@
 
 VKB_TCLFILE = $(VKBLIB_DIR)/vkeybd.tcl
 
-CFLAGS = -Wall -O -DVKB_TCLFILE=\"$(VKB_TCLFILE)\" \
+CFLAGS = $(OPTFLAGS) \
+	-Wall -DVKB_TCLFILE=\"$(VKB_TCLFILE)\" \
 	-DVKBLIB_DIR=\"$(VKBLIB_DIR)\"\
 	-DVERSION_STR=\"$(VERSION)\"\
 	$(DEVICES) $(XINC) $(TCLINC) $(TKINC) $(LADCCACFLAGS)
