diff -aurN vkeybd.orig/Makefile vkeybd/Makefile
--- vkeybd.orig/Makefile	2003-01-23 16:41:33.000000000 +0100
+++ vkeybd/Makefile	2003-10-01 14:17:45.401188432 +0200
@@ -8,7 +8,7 @@
 #
 # installation directory
 #
-PREFIX = /usr/local
+PREFIX = /usr
 # binary and Tcl script are put there
 BIN_DIR = $(PREFIX)/bin
 # man page
@@ -33,9 +33,9 @@
 # Tcl/Tk library -- depends on your distribution
 #
 TCL_VERSION = 8.4
-TCLLIB = -ltcl$(TCL_VERSION)
+TCLLIB = -ltcl
 TCLINC =
-TKLIB = -ltk$(TCL_VERSION)
+TKLIB = -ltk
 TKINC =
 XLIB = -L/usr/X11R6/lib -lX11
 XINC = -I/usr/X11R6/include
@@ -81,7 +81,8 @@
 
 VKB_TCLFILE = $(VKBLIB_DIR)/vkeybd.tcl
 
-CFLAGS = -Wall -O -DVKB_TCLFILE=\"$(VKB_TCLFILE)\" \
+CFLAGS = $(OPTFLAGS) \
+	-Wall -DVKB_TCLFILE=\"$(VKB_TCLFILE)\" \
 	-DVKBLIB_DIR=\"$(VKBLIB_DIR)\"\
 	-DVERSION_STR=\"$(VERSION)\"\
 	$(DEVICES) $(XINC) $(TCLINC) $(TKINC) $(LADCCACFLAGS)
